.travis.yml

```diff
@@ -39,7 +39,7 @@ matrix:
         # 2.7 Old dependencies
         - os: linux
           env: PYTHON_VERSION=2.7
-               CONDA_DEPENDENCIES="numpy=1.8 scipy=0.12 matplotlib=1.3 pandas=0.13 scikit-learn=0.15 pytest pytest-cov"
+               CONDA_DEPENDENCIES="numpy=1.9 scipy=0.14 matplotlib=1.4 pandas=0.16 scikit-learn=0.15 pytest pytest-cov"
```

Comment:
ahh you changed the deps. If you follow the `.travis.yml` order of operations locally you can probably reproduce the failure.

But we shouldn't do this. We should leave numpy and scipy where they are so we continue to test them

Reply:
yes, :)

thats what I was referring here https://github.com/mne-tools/mne-python/pull/5699#issuecomment-437985230 `Series.dt` was only introduced in 0.16 . 

So if we want to use it here, we require pandas 0.16.

I've been tempted to just skip, this the braking tests. And wait for the release that we drop 2.7